examples/bert/bert_train.py

```diff
@@ -189,17 +189,22 @@ def _gather_indexes(self, sequence_tensor, positions):
         return output_tensor
 
 
-class BertPretrainer(keras.Model):
-    def __init__(self, bert_model, **kwargs):
+class BertPretrainingModel(keras.Model):
+    """
```

Comment:
nit: I think this style docstring can be a single line. """MLM + NSP model with BertEncoder."""